### PR TITLE
gatsby-plugin-robots-txt

### PR DESCRIPTION
**Pull request by scottyeck**
Super simple plugin that we've been using to generate a `robots.txt` file alongside our build.

### CODE REVIEW

File: packages/gatsby-plugin-robots-txt/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# gatsby-plugin-robots-txt
+
+Provides support for writing a `robots.txt` file alongside your build with
+[robots-generator](https://www.npmjs.com/package/robots-generator).
+
+React Helmet is a component which lets you control your document head using
```
Comment (alampros) [inline packages/gatsby-plugin-robots-txt/README.md]: Is this leftover from boilerplate?

Comment (scottyeck) [reply packages/gatsby-plugin-robots-txt/README.md]: Welp good lookin out. Copied the README over. I'll kill that rn.

**Comment by KyleAMathews** [comment]
Curious why you'd do this over having a robots.txt in your static folder?

**Comment by scottyeck** [comment]
Huh. Aaaaaand turns out I totally misunderstood the use of `/static`. Yeah there's no reason for this lol.

**Comment by th0th** [comment]
It'd be awesome to be able to use different `robots.txt` files for different environments. E.g. I want to add a `robots.txt` to my repository but I don't want my development or staging environment builds to be indexed by search engines :/

**Comment by dw2** [comment]
@scottyeck I agree with what @th0th is suggesting. This plugin would be wonderful for makes environment based robots.txt files.

**Pull request closed without merging**